oidc-controller/static/jwks.json

```diff
@@ -0,0 +1 @@
+{"keys": [{"kty": "RSA", "use": "sig", "kid": "WFRiZWQ3Ukd5WU5NWF91T1ZNbWFrNmxzRGc5NHBXak1WaC1RT2ZNc21kNA", "n": "lbyeZP9VuwbP9Mu7SBsF9mi5YACKGyiqbv5BMiJHzzm_QltRJ78raMPdkjaOxSK5lYUhmG-T1boJP-4EgRPONvky_RsXqWKxqptxersbjO-6lBnbEb9usqH0WR9EgQw929pyae1YZtoNwwlljPPETzVzOog3nzHHcl6SjT63QVwJ0olu9rTcChfQZapj6h4xYdckZ9Y0uKR9Ay0ohGq_IjilA6PbMhwNt2FSH4-IHP6YiSSu39mxvaGxB67etvUYtI7re7c85X3a9K4XwJHrxmIZ-ViHZIk1m_VdL4GeZ7i1jh5B57c17lhTr9U6bRY807YsG_tgcnyIEBUY-9fVqQ", "e": "AQAB"}, {"kty": "EC", "use": "sig", "kid": "M3RmNGlQTU9EZzM3X2JWdVp2NW42S1pydzJHV2UxcklZZER3c3duSldDNA", "crv": "P-256", "x": "G0Ik-KmuzzauKFE0TWdevM8NkM2ULfL5KMXk56ZAF7E", "y": "8uaeYerNnoq2JphSEsuDzLDu4qzBFzAUK5FvJgnSFTI"}]}
```

Comment:
This file should be .gitingored: the framework should generates it on first start, if not available.